gen/nvml/device.go

```diff
@@ -1100,6 +1100,17 @@ func (Device Device) GetViolationStatus(PerfPolicyType PerfPolicyType) (Violatio
 	return DeviceGetViolationStatus(Device, PerfPolicyType)
 }
 
+// nvml.DeviceGetIrqNum()
+func DeviceGetIrqNum(Device Device) (int, Return) {
```

Comment:
I try and use `int` when its something countable, and onyl use the `uint` types when its actually required as part of the type. In 11.3 the type was referring to a megahertz value so it made since to leave it unsigned. Here we are just passing back a countable number.

Reply:
OK. I'll also try to keep that on mind when making these changes in future